pkg/ccl/changefeedccl/event_processing.go

```diff
@@ -104,7 +105,14 @@ func newEventConsumer(
 	makeConsumer := func(s EventSink, frontier frontier) (eventConsumer, error) {
 		sourceData := enrichedSourceData{}
 		if encodingOpts.Envelope == changefeedbase.OptEnvelopeEnriched {
-			sourceData, err = newEnrichedSourceData(ctx, cfg, spec, sink.getConcreteType())
+			schemaInfo := make(map[descpb.ID]tableSchemaInfo)
```

Comment:
not: prefer var declaration here, unless empty vs nil is meaningful to your code, which it shouldn't be ideally